mautic/mautic
# Remove iframeStyleFiles option from froala

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

#### Description:
The iframeStyleFiles option prevents CSS links in the page <head> from saving correctly. This PR removes our setting.

#### Steps to test this PR:
1. Apply PR
2. Check to ensure gated video still works and that the video boxes are styled properly on insert.
